webapp/channels/src/components/emoji_picker/constants/index.ts

```diff
@@ -125,9 +125,10 @@ export const CATEGORIES: Categories = Emoji.CategoryNames.
         };
     }, {} as Categories);
 
-export const EMOJI_PER_ROW = 9; // needs to match variable `$emoji-per-row` in _variables.scss
-export const ITEM_HEIGHT = 36; //as per .emoji-picker__item height in _emoticons.scss
-export const EMOJI_CONTAINER_HEIGHT = 290; // If this changes, the spaceRequiredAbove and spaceRequiredBelow props passed to the EmojiPickerOverlay must be updated
+export const EMOJI_PER_ROW = 9;
+export const ITEM_HEIGHT = 36;
+export const EMOJI_CONTAINER_HEIGHT = 290;
```

Comment:
Could you re-add the existing comments?
```suggestion
export const EMOJI_PER_ROW = 9; // needs to match variable `$emoji-per-row` in _variables.scss
export const ITEM_HEIGHT = 36; //as per .emoji-picker__item height in _emoticons.scss
export const EMOJI_CONTAINER_HEIGHT = 290; // If this changes, the spaceRequiredAbove and spaceRequiredBelow props passed to the EmojiPickerOverlay must be updated
```